proxyservice/dockerhub.go

```diff
@@ -54,8 +56,10 @@ type DockerHubWebhookData struct {
 
 // Callback calls data's callback_url
 func (d *DockerHubWebhookData) Callback(cb *CallBackData) error {
-	if !strings.HasPrefix(d.CallbackURL, "https://registry.hub.docker.com/") {
-		return errors.New("d.CallBackURL does not start with https://registry.hub.docker.com/")
+	callbackPrefix := fmt.Sprintf("%s/u/%s/%s/hook/",
+		DockerhubRegistry, d.Repository.Namespace, d.Repository.Name)
+	if !strings.HasPrefix(d.CallbackURL, callbackPrefix) {
```

Comment:
The hook contains a random hash after `/hook/`, something like `/hook/2141b5bi5i5b02bec211i4eeih0242eg11000a/`